docs/guides/validation-user-guide.md

```diff
@@ -4,4 +4,461 @@ headline: Validation User Guide
 bodyclass: docs
 layout: docs
 ---
-<p class="coming-soon">Coming soon...</p>
+
+Protobuf is a great tool for structuring domain models. Protobuf definitions help us to standardize
+creating [Value Objects](https://martinfowler.com/bliki/ValueObject.html).
+Spine enhances Protobuf with a validation library.
```

Comment:
I don't think we extend the language or the library. I'd say that the framework provide the Validation library which builds on top of Protobuf options and plugins to Protobuf compiler.